cmake/module.cmake

```diff
@@ -199,6 +199,11 @@ function(fprime__internal_add_build_target_helper TARGET_NAME TYPE SOURCES AUTOC
     else()
         fprime_cmake_fatal_error("Cannot register compilation target of type ${TYPE}")
     endif()
+    if (TYPE STREQUAL "Unit Test" AND INTERNAL_UT_AUTO_HELPERS)
+        set_target_properties("${TARGET_NAME}" PROPERTIES 
+            INTERNAL_UT_AUTO_HELPERS TRUE
```

Comment:
Name this FPRIME_UT_AUTO_HELPERS.  INTERNAL is only the prefix for the current scope (this function).  Custom Fprime target properties are named FPRIME_UT_AUTO_HELPERS.

Reply:
Fixed